plugins/modules/apt_rpm.py

```diff
@@ -155,25 +210,51 @@ def main():
         argument_spec=dict(
             state=dict(type='str', default='present', choices=['absent', 'installed', 'present', 'removed']),
             update_cache=dict(type='bool', default=False),
-            package=dict(type='list', elements='str', required=True, aliases=['name', 'pkg']),
+            clean=dict(type='bool', default=False),
+            dist_upgrade=dict(type='bool', default=False),
+            update_kernel=dict(type='bool', default=False),
+            package=dict(type='list', elements='str', aliases=['name', 'pkg']),
         ),
     )
 
     if not os.path.exists(APT_PATH) or not os.path.exists(RPM_PATH):
         module.fail_json(msg="cannot find /usr/bin/apt-get and/or /usr/bin/rpm")
 
     p = module.params
+    modified = False
+    output = ""
 
     if p['update_cache']:
-        update_package_db(module)
+        (m, out) = update_package_db(module)
+        modified = modified or m
```

Comment:
This is a breaking change that will prevent merging this PR. Please create a new PR for this part, and remove it from this PR. (See #3907 and #4318)